ding/torch_utils/reshape_helper.py

```diff
@@ -0,0 +1,55 @@
+from typing import Tuple, Union
+
+from torch import Tensor, Size
+
+
+def fold_batch(x: Tensor, nonbatch_dims=1) -> Tuple[Tensor, Size]:
```

Comment:
why use `dims` to refer to a integer

Reply:
changed to `ndims` following `torch.distributions.Independent`.